modules/adding-tls-termination.adoc

```diff
@@ -0,0 +1,60 @@
+// Module included in the following assemblies:
+//
+// * networking/aws_load_balancer_operator/add-tls-termination.adoc
+
+:_content-type: PROCEDURE
+[id="nw-adding-tls-termination_{context}"]
+= Adding the tls termination on the AWS Load Balancer
+
+You can route the traffic for the domain `example.com` to pods of a `service` `example-service`, and add TLS termination on the AWS Load Balancer.
+
+.Prerequisites
+
+.Procedure
+
+. Deploy the operator and create an instance of the resource `AWSLoadBalancerController`:
++
+[source,yaml]
+----
+kind: AWSLoadBalancerController
+group: networking.olm.openshift.io/v1alpha1 <1>
+metadata:
+  name: cluster <2>
+spec:
+  subnetTagging: auto
+  ingressClass: tls-termination <3>
+----
+
+<1> Defines the API group of the `AWSLoadBalancerController` resource.
+<2> Defines the `aws-load-balancer-controller` instance.
+<3> The name of the `IngressClass` which the `aws-load-balancer-controller` will reconcile. This `IngressClass` will be created if it is not present. Additional `IngressClass`es can also be added which will be reconciled by the controller as long as their `spec.controller` is `ingress.k8s.aws/alb`.
+
+. Create an `Ingress` resource:
++
+[source,yaml]
+----
+apiVersion: networking.k8s.io/v1
+kind: Ingress
+metadata:
+  name: example
+  annotations:
+    alb.ingress.kubernetes.io/scheme: internet-facing <1>
+    alb.ingress.kubernetes.io/certificate-arn: arn:aws:acm:us-west-2:xxxxx <2>
+spec:
+  ingressClassName: tls-termination <3>
+  rules:
+  - host: example.com
+    http:
+        paths:
+          - path: /
+            pathType: Exact
+            backend:
+              service:
+                name: example-service
+                port:
+                  number: 80
+----
+
+<1> Controller provisions the load balancer for this ingress in a public subnet so that the load balancer is reachable over the internet.
```

Comment:
The controller provisions the load balancer for this `Ingress` resource

Reply:
Ack'ed